doc/gh-pages/pages/download/development_versions.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Development Versions
+parent: Download
+nav_order: 1
+---
+# Development Versions
+
+For older releases please visit our [Github release page](https://github.com/mne-tools/mne-cpp/releases){:target="_blank" rel="noopener"}. If you want to checkout more recently added features, you can download the development versions of the application.

```

Comment:
'Checkout' should be 'check out', unless this is a git pun, in which case i'm all for it.

Reply:
It was not. Perhaps it should be. But just bad spelling.